.travis.yml

```diff
@@ -35,7 +35,6 @@ before_install:
 - bash scripts/setup_couch.sh
 
 install:
```

Comment:
Can we remove the whole `install` block as we don't appear to be installing anything else?
```suggestion
```

Reply:
Done in 0c62f16.